home.css

```diff
@@ -0,0 +1,674 @@
+@font-face {
+  font-family: cocogoose;
+  src: url(fonts/Cocogoose-font.otf);
+}
+
+html,
+body {
+  height: 100%;
+  padding: 0;
+  margin: 0;
+  font-size: 16px;
+  font-family: "lato", sans-serif;
+}
+
+.event-details {
+  padding-bottom: 10%;
+}
+
+p {
+  line-height: 1.1rem;
+}
+
+.menu-content,
+.top-bar-section,
+.close-menu,
+.logo-desktop a {
+  display: none;
+}
+
+.top-bar-section {
+  background-color: #3e3e3e;
+}
+
+.navigation-container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: center;
+  width: 112%;
+}
+
+.nav-toggle {
+  position: absolute;
+  top: 1rem;
+  cursor: pointer;
+  left: 1rem;
+}
+
+.close-menu {
+  cursor: pointer;
+  z-index: 10;
+}
+
+.close-menu p {
+  font-size: 3.2rem;
+  position: absolute;
+  margin: 0;
+  top: 1.3rem;
+  left: 1.3rem;
+  color: black;
+  z-index: 100;
+}
+
+.logo-desktop a {
+  position: absolute;
+  top: 1.3rem;
+  right: 1.5rem;
+  text-decoration: none;
+  color: black;
+}
+
+.pages-menu {
+  width: 100vw;
+  display: none;
+  height: 100vh;
+  background-color: rgba(234, 234, 234, 0.9);
+  position: absolute;
+  z-index: 5;
+  top: 0;
+  left: 0;
+  border-right: 4px outset #ec5242;
+  border-bottom: 2px outset #ec5242;
+}
+
+.open {
+  height: 100vh;
+}
+
+.menu-items {
+  padding: 0;
+  margin-top: 8rem;
+}
+
+.menu-items li {
+  display: block;
+  text-align: center;
+  padding: 1.5rem 0;
+}
+
+.web-pages a {
+  text-decoration: none;
+  color: black;
+  font-size: 1.3rem;
+}
+
+.home-page a {
+  text-decoration: none;
+  color: black;
+  font-size: 1.3rem;
+}
+
+.company-name {
+  display: none;
+}
+
+.company-name a {
+  text-decoration: none;
+  color: black;
+}
+
+.footer-logo a {
+  font-size: 2rem;
+  font-weight: 800;
+  text-decoration: none;
+  color: black;
+}
+
+.web-pages a:hover {
+  color: #ec5242;
+  border-bottom: 1px solid #ec5242;
+  background-color: #f7f7f8;
+  font-size: 1.2rem;
+  padding-inline: 11%;
+  padding-block: 5%;
+}
+
+.home-page a:hover {
+  color: #ec5242;
+  border-bottom: 1px solid #ec5242;
+  background-color: #f7f7f8;
+  font-size: 1.2rem;
+  padding-inline: 11%;
+  padding-block: 5%;
+}
+
+.fa-bars {
+  font-size: 2rem;
+}
+
+.first-section {
+  background-image: url(assets/nba9.jpg);
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
- [ ] Also the background image is kind of not fitting in the mobile view
![bb](https://user-images.githubusercontent.com/61562626/129460400-56e92b18-672e-4831-b221-b83309729ca0.jpeg)